edb/pgsql/compiler/dml.py

```diff
@@ -440,7 +440,7 @@ def compile_insert_shape_element(
             # anyway.
             insvalctx.volatility_ref = context.NO_VOLATILITY
 
-        insvalctx.path_scope[ir_stmt.subject.path_id] = insert_stmt
+        # insvalctx.path_scope[ir_stmt.subject.path_id] = insert_stmt
```

Comment:
Is this supposed to stay commented out?

Reply:
It's supposed to be deleted, which I've now done.